build.gradle

```diff
@@ -1,8 +1,8 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = "1.4.32"
+    ext.kotlin_version = "1.5.21"
```

Comment:
In this version, the capitalize() method has been deprecated. You could fix the warning in the DetailCharacterFragment class